src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertFromJsonCommand.cs

```diff
@@ -43,6 +42,13 @@ public class ConvertFromJsonCommand : Cmdlet
         [ValidateRange(ValidateRangeKind.Positive)]
         public int Depth { get; set; } = 1024;
 
+        /// <summary>
+        /// Prevents ConvertFrom-Json from unravelling collections during deserialization, instead passing them as a single
```

Comment:
```suggestion
        /// Gets or sets the switch to prevent ConvertFrom-Json from unravelling collections during deserialization, instead passing them as a single
```